public/app/features/plugins/sandbox/code_loader.ts

```diff
@@ -19,8 +19,13 @@ export async function loadScriptIntoSandbox(url: string, meta: PluginMeta, sandb
   if (isSameDomainAsHost(url)) {
     const response = await fetch(url);
     scriptCode = await response.text();
-    scriptCode = patchPluginSourceMap(meta, scriptCode);
-
+    //even though this is not loaded via a CDN we need to transform the sourceMapUrl
+    scriptCode = transformPluginSourceForCDN({
+      url,
+      source: scriptCode,
+      transformSourceMapURL: true,
+      transformAssets: false,
+    });
```

Comment:
Do we also need to do it in `getPluginCode()` [here](https://github.com/grafana/grafana/blob/677a2ad4d8a634488508c22102dd8fb3096a95c9/public/app/features/plugins/sandbox/code_loader.ts#L60)?

Reply:
Good idea. I'll replace that code with this one